composer.json

```diff
@@ -1,16 +1,29 @@
 {
     "name": "tightenco/quicksand",
-    "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*"
-    },
-    "license": "MIT",
+    "description": "Easily schedule regular cleanup of old soft-deleted Eloquent data.",
     "authors": [
         {
             "name": "Benson Lee",
             "email": "benson@tighten.co"
+        },
+        {
+            "name": "Matt Stauffer",
+            "email": "matt@tighten.co"
         }
     ],
+    "license": "MIT",
+    "require": {
+        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*"
+    },
+    "require-dev": {
+        "illuminate/database": "5.*",
+        "nesbot/carbon": "^1.21",
+        "mockery/mockery": "^0.9.5"
+    },
     "autoload": {
+        "psr-0": {
```

Comment:
Any reason we're not doing classmap instead? Never seen a psr-0 autoload on `tests` before